### PR TITLE
Merge Iter branch

### PR DESCRIPTION
**Pull request by behdad**
Makes C++11 a requirement.

### CODE REVIEW

**Comment by behdad** [comment]
cc @garretrieger

**Comment by behdad** [comment]
@ebraminio any idea why this bot is failing: https://circleci.com/gh/harfbuzz/harfbuzz/79320?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: .circleci/config.yml [hunk 1]
```diff
@@ -92,7 +80,7 @@ jobs:
 
   clang-O3-O0:
     docker:
-      - image: multiarch/crossbuild
+      - image: ubuntu:18.10
```
Comment (ebraminio) [inline .circleci/config.yml]: Please add clang to this bot also

**Comment by ebraminio** [comment]
> @ebraminio any idea why this bot is failing:

Clang was not available on the new image and easily can be added

File: .circleci/config.yml (hunk 1 above)
Comment (behdad) [reply .circleci/config.yml]: Weird.  So gcc wasn't providing a compiler?

Comment (behdad) [reply .circleci/config.yml]: The bot is named gcc-valgrind though.  Why do we need clang?

Comment (behdad) [reply .circleci/config.yml]: Ooh my bad.

**Comment by behdad** [comment]
Merged manually...

**Pull request merged**